### ISSUE TITLE
ItemLoader.UseItemFrame doesn't get called.

### ISSUE THREAD

**Issue by Liburia** [tModLoader/tModLoader#319]
### Description

ItemLoader.UseItemFrame doesn't get called if item.useStyle is 1, 2, 3, 4 or 5.

### Expected behavior 

Called regardless of item.useStyle.

### Actual behavior

```
if (this.inventory[this.selectedItem].useStyle == 5)
{
    ...
}
else if (ItemLoader.UseItemFrame(this.inventory[this.selectedItem], this))
{
    return;
}
```

### Reproduction frequency

Every time.

### Versions

`tModLoader v0.10.1.3`

**Comment by Jofairden** [comment]
It would be useful if you could provide example files that reproduce this issue. I can test if you provide them.

**Comment by Jofairden** [comment]
Since no further comment is given, I assume the problem is solved. Feel free to reply if it isn't.

**Comment by JavidPack** [comment]
This one should stay open actually, I looked into it and it does seem a bit odd that ItemLoader.UseItemFrame can't affect the original useStyles.

**Comment by JavidPack** [comment]
I think I'll wait until I hear more support for calling this regardless. The design of the hook as it currently is seems to assume that only 1 method will take effect, since it short circuits if any global item or moditem returns true. 

If someone can think of a good reason to call a GlobalItem.UseItemFrame regardless, that would help us decide.

**Comment by Chicken-Bones** [comment]
Needs a usecase

**Comment by Jofairden** [comment]
No activity/further reports.

**Comment by JavidPack** [comment]
Yeah, we need to fix this in 1.4

**Comment by Mirsario** [comment]
Fixed in https://github.com/tModLoader/tModLoader/commit/f850a3b9432a10bfee50c965b7e9476f9186c60c
